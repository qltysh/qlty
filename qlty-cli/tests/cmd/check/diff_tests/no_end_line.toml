bin.name = "qlty"
args = ["check", "--no-cache", "--verbose"]
status.code = 1
