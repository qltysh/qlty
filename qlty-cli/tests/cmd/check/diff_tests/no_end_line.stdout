
 ISSUES: 1 

diff/test_sample.rb:1:0
    1:0  medium  undefined name 'foo'[..]no_end_line:F821

 JOBS: 1 

Plugin       Result   Targets   Time   Debug File
no_end_line  Success  1 target  [..]s  .qlty/out/invoke-[..].yaml

Checked 1 modified file
